litmus/litmus.c

```diff
@@ -520,7 +520,6 @@ int switch_sched_plugin(struct sched_plugin* plugin)
 	struct domain_proc_info* domain_info;
 
 	BUG_ON(!plugin);
-
```

Comment:
Let's try to avoid whitespace changes - we want to change as little as necessary to fix upstream.